services/categories.js

```diff
@@ -9,3 +9,14 @@ exports.getCategories = async () => {
     throw new ErrorObject(error.message, error.statusCode || 500)
   }
 }
+
+exports.updateCategory = async (id, updateInfo) => {
+  try {
+    const findCategory = await Category.findByPk(id)
+    if (!findCategory) throw new ErrorObject('this category does not exist', 404)
```

Comment:
No estoy seguro de esto, pero viendo los PRs anteriores, vi que mencionaban que el IF no hace falta, pues si no encuentra el ID, sale x el  catch del try., podes probar en todo caso.

Reply:
si le saco el if tendria que personalizar el error del catch porque necesito dar un error 404 "not found" para decir que no existe la categoria que se quiere updatear, y el catch es 500 que es un codigo de error generico, el tema que si lo cambio ya no quedaria un control de errores generales